locustfile.py

```diff
@@ -10,13 +10,14 @@
 from locust import HttpLocust, TaskSet, task
```

Comment:
It might be nice to add to a docstring explaining that you need to create a user with a username of `USERNAME` and password of `PASSWORD`.